packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -282,7 +281,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     title: css({
       label: 'panel-title',
       marginBottom: 0, // override default h6 margin-bottom
-      paddingRight: theme.spacing(1),
+      padding: theme.spacing(0, padding),
```

Comment:
Padding for the whole header created a gap that can be used for dragging before the elements if there is no title: 
Before - 
<img width="501" alt="image" src="https://user-images.githubusercontent.com/13227501/221801121-d801f5c5-7797-41c8-9c94-742645d81500.png">

After -
<img width="496" alt="image" src="https://user-images.githubusercontent.com/13227501/221800861-ea758ed9-086a-4107-b30e-564871d8c541.png">

Reply:
There should be no padding to the left of the leftmost item , otherwise, so that it's snug in the corner

Reply:
You can drag in the whole are between title items and the menu no? I don't think the 8px in the left corner is important

Reply:
Yes, I'm just explaining the [commit](https://github.com/grafana/grafana/pull/63776/commits/5df18b29963db7ca4437351981c3a8528ed5cb9d) I made 🙈.